### PR TITLE
Add Documentation for the JSM Integration

### PR DESCRIPTION
**Pull request by rain-on**
Copy and paste of the ServiceNow docs - but search/replaced servicenow for JSM (but not really).

### CODE REVIEW

File: docs/approvals/jira-service-management/index.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+control using Change Requests/Issues
+position: 10
+---
+
+:::warning
+The Jira Service Management (JSM) Integration feature is only available as an EAP for version 
+**2022.3.
+12101** or later and it requires a specific feature license to use it. To request a 
+license register for the [JSM Early Access Program](https://octopusdeploy.
+typeform.com/servicenow-eap)
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:13]: should be a link to the JSM eap, but you probably already were aware of this

File: docs/approvals/jira-service-management/index.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+The integration will require a user account in JSM. The recommendation is to create a 
+service account specifically for Octopus.
+
+Take note of the password assigned or generated for this user.
+
+### Licensing
+
+For the JSM approval checks to be performed as part of the deployment process, an appropriate Octopus license must be configured in your Octopus instance.
+
+A JSM enabled Octopus license must be requested from Octopus directly, and cannot be managed through the self-service process. To request a license register for the [JSM Early Access Program](https://octopusdeploy.typeform.com/servicenow-eap)
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:60]: https://github.com/OctopusDeploy/docs/pull/1636/files#r956942139

File: docs/approvals/jira-service-management/index.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+:::hint
+The instructions in this section will require an Octopus Deploy Manager or Administrator
+:::
+
+To connect your Octopus Deploy instance to JSM, navigate to **Configuration ➜ Settings ➜ 
+Jira Service Management Integration**.
+
+Check the **Enabled** option
+![ServiceNow Integration Settings page](images/jsm-connections-1.png "width=500")
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:84]:
```suggestion
![Jira Service Management Integration Settings page](images/jsm-connections-1.png "width=500")
```

File: docs/approvals/jira-service-management/index.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+Click on **ADD CONNECTION** and fill out the details.
+The JSM Base Url should be the root URL and include the protocol e.g. `https://`
+
+![ServiceNow Integration Add Connection](images/jsm-connections-2.png "width=500")
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:89]:
```suggestion
![Jira Service Management Integration Add Connection](images/jsm-connections-2.png "width=500")
```

File: docs/approvals/jira-service-management/index.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+The instructions in this section will require an Octopus Deploy Manager or Administrator
+:::
+
+If enabled, this feature will result in a linked change request having one or more Comments added 
+during the deployment lifecycle which record details about the deployment and its execution status.
+
+To enable this feature navigate to **Configuration ➜ Settings ➜ Jira Service Management 
+Integration**, click the **Customer Comments Enabled** checkbox show below then click **Save**.
+
+![ServiceNow Integration Enable Work Notes](images/jsm-worknotes-settings.png "width=500")
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:108]:
```suggestion
![Jira Service Management Integration Enable Work Notes](images/jsm-worknotes-settings.png "width=500")
```

File: docs/approvals/jira-service-management/index.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+### Setting up projects for CR approval
+
+To enable a project to enforce a requirement for an approved CR:
+
+1. Navigate to the project and then **{{Settings,General}}**.
+2. Check the **Jira Service Management Integration ➜ Change Controlled** setting.
+3. Select your JSM connection in the **Jira Service Management Connection** setting and click 
+   **SAVE**.
+
+![ServiceNow Integration Project settings](images/jsm-project-settings.png "width=500")
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:123]:
```suggestion
![Jira Service Management Integration Project settings](images/jsm-project-settings.png "width=500")
```

File: docs/approvals/jira-service-management/index.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+then an Issue will not be created, and instead, the supplied number will be used during the 
+approval check. This variable can also be [scoped](/docs/projects/variables/index.md#scoping-variables) or configured as a [Prompted variable](/docs/projects/variables/prompted-variables.md).
+
+### Setting up environments for CR approval
+
+To enable an environment to enforce a requirement for an approved CR, navigate to **{
+{Infrastructure,Environments}}**, edit the environment via the overflow menu and check the 
+**Jira Service Management Integration ➜ Change Controlled** setting, and then press **SAVE**.
+
+![ServiceNow Integration Environment settings](images/jsm-environment-settings.png "width=500")
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:142]:
```suggestion
![Jira Service Management Integration Environment settings](images/jsm-environment-settings.png "width=500")
```

File: docs/approvals/jira-service-management/index.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+
+:::info
+The only supported states are those defined in the default Issue lifecycle
+:::
+
+The number of the Issue created or found will appear in the Task Summary tab of the executing 
+Octopus deployment task. Clicking on the CR number in the message will navigate you to the CR in 
+JSM.
+
+![Deployment Task Summary awaiting ServiceNow approval](images/jsm-pending-issue-task-message.png
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:174]:
```suggestion
![Deployment Task Summary awaiting Jira Service Management approval](images/jsm-pending-issue-task-message.png 
```

File: docs/approvals/jira-service-management/index.md (hunk 1 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md:13]: add a task to the ticket for that - but need to work out what to do about that first!

File: docs/approvals/jira-service-management/index.md [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+
+:::hint
+The instructions in this section will require an Octopus Deploy Manager or Administrator
+:::
+
+To connect your Octopus Deploy instance to JSM, navigate to **Configuration ➜ Settings ➜ 
+Jira Service Management Integration**.
+
+Check the **Enabled** option
+![JSM Integration Settings page](images/jsm-connections-1.png "width=500")
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:84]: Can this image be made narrower, there is a lot of whitespace

File: docs/approvals/jira-service-management/index.md [hunk 10]
```diff
@@ -0,0 +1,211 @@
...
+then an Issue will not be created, and instead, the supplied number will be used during the 
+approval check. This variable can also be [scoped](/docs/projects/variables/index.md#scoping-variables) or configured as a [Prompted variable](/docs/projects/variables/prompted-variables.md).
+
+### Setting up environments for CR approval
+
+To enable an environment to enforce a requirement for an approved CR, navigate to **{
+{Infrastructure,Environments}}**, edit the environment via the overflow menu and check the 
+**Jira Service Management Integration ➜ Change Controlled** setting, and then press **SAVE**.
+
+![JSM Integration Environment settings](images/jsm-environment-settings.png "width=500")
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:142]: Can this image be made narrower, there is a lot of whitespace

File: docs/approvals/jira-service-management/index.md [hunk 11]
```diff
@@ -0,0 +1,211 @@
...
+### Setting up projects for CR approval
+
+To enable a project to enforce a requirement for an approved CR:
+
+1. Navigate to the project and then **{{Settings,General}}**.
+2. Check the **Jira Service Management Integration ➜ Change Controlled** setting.
+3. Select your JSM connection in the **Jira Service Management Connection** setting and click 
+   **SAVE**.
+
+![JSM Integration Project settings](images/jsm-project-settings.png "width=500")
```
Comment (benPearce1) [inline docs/approvals/jira-service-management/index.md:123]: Can this image be made narrower, there is a lot of whitespace

**Review by benPearce1 — CHANGES_REQUESTED** [review]
Some of the images need the whitespace removed.

File: docs/approvals/jira-service-management/index.md [hunk 12]
```diff
@@ -0,0 +1,211 @@
+---
+title: Jira Service Management Integration
+description: Octopus Deploy can integrate with your Jira Service Management instance for deployment 
+control using Change Requests/Issues
+position: 10
+---
+
+:::warning
+The Jira Service Management (JSM) Integration feature is only available as an EAP for version 
+**2022.3.
```
Comment (long-sean) [inline docs/approvals/jira-service-management/index.md:10]: nit: line break location

File: docs/approvals/jira-service-management/index.md [hunk 13]
```diff
@@ -0,0 +1,211 @@
...
+1. Configure a connection from Octopus to JSM.
+1. Configure which deployments require an approved CR.
+
+### Configuring Jira Service Management
+
+:::hint
+The instructions in this section will require a JSM Administrator.
+:::
+
+The Octopus Deploy / JSM integration requires security configuration in your target JSM instance.
```
Comment (long-sean) [inline docs/approvals/jira-service-management/index.md:49]: nit: on line 18 and 36 we don't put spaces around the `/` but we do here (consistency)

File: docs/approvals/jira-service-management/index.md [hunk 14]
```diff
@@ -0,0 +1,211 @@
...
+:::
+
+To connect your Octopus Deploy instance to JSM, navigate to **Configuration ➜ Settings ➜ 
+Jira Service Management Integration**.
+
+Check the **Enabled** option
+![JSM Integration Settings page](images/jsm-connections-1.png "width=500")
+
+Click on **ADD CONNECTION** and fill out the details.
+The JSM Base Url should be the root URL and include the protocol e.g. `https://`
```
Comment (long-sean) [inline docs/approvals/jira-service-management/index.md:87]: nit: line break

**Review by long-sean — COMMENTED** [review]
Only a few nits from me, also I do agree with @benPearce1 regarding the whitespace in images

File: docs/approvals/jira-service-management/index.md (hunk 4 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md]: Done

File: docs/approvals/jira-service-management/index.md (hunk 5 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md]: Done

File: docs/approvals/jira-service-management/index.md (hunk 6 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md]: Done

File: docs/approvals/jira-service-management/index.md (hunk 7 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md]: Done

File: docs/approvals/jira-service-management/index.md (hunk 8 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md]: Done

File: docs/approvals/jira-service-management/index.md (hunk 11 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md:123]: updated

File: docs/approvals/jira-service-management/index.md (hunk 9 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md:84]: updated

File: docs/approvals/jira-service-management/index.md (hunk 12 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md:10]: done

File: docs/approvals/jira-service-management/index.md (hunk 13 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md:49]: done

File: docs/approvals/jira-service-management/index.md (hunk 14 above)
Comment (rain-on) [reply docs/approvals/jira-service-management/index.md:87]: not sure what I'm missing - 87 should end on "https://" - then 89 is the next instruction

**Review by benPearce1 — APPROVED** [review]
Approved.

File: docs/approvals/index.md [hunk 15]
```diff
@@ -5,7 +5,10 @@ position: 95
 hideInThisSection: true
 ---
 
-Octopus currently supports blocking deployments for a project and environment based on the approval status of a Change Request in a supported ITSM system. Currently, only ServiceNow is supported.
+Octopus currently supports blocking deployments for a project and environment based on the 
+approval status of a Change Request in a supported ITSM system. Currently, both ServiceNow and 
+Jira Service Desk are supported.
```
Comment (harrisonmeister) [inline docs/approvals/index.md:10]:
```suggestion
Octopus Deploy includes dedicated integrations for popular ITSM systems to help support blocking deployments for a project and environment based on the approval status of a Change Request.
```

File: docs/approvals/index.md [hunk 16]
```diff
@@ -5,7 +5,10 @@ position: 95
...
 
-For more information, read our documentation on [Service Now Integration](service-now/index.md).
+For more information, read our documentation on [Service Now Integration](service-now/index.md) 
+and [Jira Service Management Integration](jira-service-management/index.md).
```
Comment (harrisonmeister) [inline docs/approvals/index.md:13]: Removing this allows for the child page selector to show the child pages for you as they expand over time :) 
```suggestion
```

File: docs/approvals/index.md [hunk 17]
```diff
@@ -5,7 +5,10 @@ position: 95
 hideInThisSection: true
```
Comment (harrisonmeister) [inline docs/approvals/index.md:5]: If you remove the section header, and not the entire contents, the page becomes more dynamic
```suggestion
hideInThisSectionHeader: true
```

File: docs/approvals/jira-service-management/index.md [hunk 18]
```diff
@@ -0,0 +1,209 @@
+---
+title: Jira Service Management Integration
+description: Octopus Deploy can integrate with your Jira Service Management instance for deployment control using Change Requests/Issues
+position: 10
+---
+
+:::warning
+The Jira Service Management (JSM) Integration feature is only available as an EAP for version
```
Comment (harrisonmeister) [inline docs/approvals/jira-service-management/index.md:8]:
```suggestion
The Jira Service Management (JSM) Integration feature is only available as an EAP for Octopus 
```

File: docs/approvals/jira-service-management/index.md [hunk 19]
```diff
@@ -0,0 +1,209 @@
+---
+title: Jira Service Management Integration
+description: Octopus Deploy can integrate with your Jira Service Management instance for deployment control using Change Requests/Issues
+position: 10
+---
+
+:::warning
+The Jira Service Management (JSM) Integration feature is only available as an EAP for version 
+**2022.3.12101** or later and it requires a specific feature license to use it. To request a
```
Comment (harrisonmeister) [inline docs/approvals/jira-service-management/index.md:9]:
```suggestion
**2022.3.12101** or later, and it requires a specific feature license to use it. To request a 
```

File: docs/approvals/jira-service-management/index.md [hunk 20]
```diff
@@ -0,0 +1,209 @@
...
+:::warning
+The Jira Service Management (JSM) Integration feature is only available as an EAP for version 
+**2022.3.12101** or later and it requires a specific feature license to use it. To request a 
+license register for the [JSM Early Access Program](https://octopusdeploy.typeform.com/jsm-eap)
+:::
+
+## Overview
+
+The Octopus Deploy/JSM integration allows users to block the execution of 
+specifically configured deployments unless they have a corresponding approved JSM  **Change
```
Comment (harrisonmeister) [inline docs/approvals/jira-service-management/index.md:16]:
```suggestion
specifically configured deployments unless they have a corresponding approved JSM **Change 
```

File: docs/approvals/jira-service-management/index.md [hunk 21]
```diff
@@ -0,0 +1,209 @@
...
+1. Navigate to the project and then **{{Settings,General}}**.
+2. Check the **Jira Service Management Integration ➜ Change Controlled** setting.
+3. Select your JSM connection in the **Jira Service Management Connection** setting and click 
+   **SAVE**.
+
+![JSM Integration Project settings](images/jsm-project-settings.png "width=500")
+
+### Default Behavior
+
+Deployments resulting in a CR creation will produce an issue with a Request Type of "Request a
```
Comment (harrisonmeister) [inline docs/approvals/jira-service-management/index.md:125]:
```suggestion
Deployments resulting in a CR creation will produce an issue with a Request Type of **Request a 
```

File: docs/approvals/jira-service-management/index.md [hunk 22]
```diff
@@ -0,0 +1,209 @@
...
+2. Check the **Jira Service Management Integration ➜ Change Controlled** setting.
+3. Select your JSM connection in the **Jira Service Management Connection** setting and click 
+   **SAVE**.
+
+![JSM Integration Project settings](images/jsm-project-settings.png "width=500")
+
+### Default Behavior
+
+Deployments resulting in a CR creation will produce an issue with a Request Type of "Request a 
+change".
```
Comment (harrisonmeister) [inline docs/approvals/jira-service-management/index.md:126]:
```suggestion
change**
```

**Pull request merged**